Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -216,13 +258,45 @@ public override long CountIndex(Guid chainId)
                 }
             }
 
-            return IndexCollection(chainId).FindById(index + 1)?.Hash;
+            ColumnFamilyHandle cf = GetColumnFamily(chainId);
+            byte[] indexBytes = BitConverter.GetBytes(index);
+
+            // Use Big-endian to order index lexicographically.
+            if (BitConverter.IsLittleEndian)
+            {
+                Array.Reverse(indexBytes);
+            }
```

Comment:
Instead of this kind of branch, how about using `NetworkOrderBitsConverter` provided by NetMQ? Some classes in the `Libplanet.Net.Messages` namespace have been already used this.

```suggestion
            byte[] indexBytes = NetMQ.NetworkOrderBitsConverter.GetBytes(index);
```